### PR TITLE
feat(opportunities): start the new design for the opportunities

### PR DESCRIPTION
**Pull request by Fdom92**
https://openforge.teamwork.com/#/tasks/16807279

### CODE REVIEW

**Comment by Wholloran** [comment]
All:
- "We want to put you to the test!" section needs its language and icon updated 
- "Are you prepared" text and icon needs to be updated (Notes this section is different under dev/design)
- Tablet/desktop: add additional spacing between the paragraph titles and content (Forget normal interviews, we want to put you to the test, reputation is everything, are you prepared)
- Update the slider titles for Design (the categories have changed)
when the user's presses "apply" (under the sliders) the page changes but the user is not pushed to the top of the page, they should be
- After the user navigates past the slider page, the animation at the top of the screen shows both the dev/design. per the marvel, it should only show the selected animation (dev or design)
- swap the asset under "the ideal candidate can..." for the new one in the marvel (not this asset it dependent on dev/design
-  The "apply now" button under show us what you are made of should push the user to the application form

Desktop:
- decrease the padding between the content and image for "the ideal candidate can" & "and can meet the following prerequisites". This was the text stretches further and isn't as stacked 

Tablet:
- The animation doesn't cover the full screen it's pushed to the left 
- "forget normal interviews" title and it's paragraph needs right-aligned per the marvel
- the content for the opportunities landing page needs to be lower so it's not overlapping the gears/squares from the animation
- when the user presses developer or Designer the background animation disappears (background is white) and then arrow animation is incorrect. (see screenshot)

Mobile:
- "forget normal interviews" & "Reputation is everything" titles are not centered on mobile
-The animation for when dev/design is selected needs to be centered and moved down on safari v12.4.1
- "Show us what you're made of " should be stacked, the paragraph and the button under it should be center-aligned as well.
- Add additional spacing between the apply now button and the dividing line
- Center "ideal candidate can..." & "and can meet the following prerequisites"
- move the assets from under "ideal candidate can..." to below the bullet points

Form:
- Update Resume / Cv upload to match the marvel https://marvelapp.com/3f8d0id/screen/63248698
- Remove the text above "Submit your application"
- add place holder text for all fields
- change github url to "link to your profile when under the design flow
- add "*" to the end of the required field titles

**Comment by Fdom92** [comment]
@Wholloran I updated the PR with lot of fixes for that feedback, but, all the assets or animation related I'm still waiting for them so I can not fix those ones yet

**Comment by Wholloran** [comment]
sounds good. I'll do another pass to check the other fixes you implemented

**Comment by Wholloran** [comment]
All:
- Change the language under normal interview suck for the **design** side. that section should read: "At OpenForge, we believe that sometimes actions speak louder than words, so as part of your interview assignment, we’d like to challenge you to design 3 pages of an app of your choosing."
-  Have "Apply Now" scroll less (should show "submit your application" )
- Still need assets swapped. if you still don't have the assets let me know tomorrow and I'll get the team to hand them off asap.

Desktop:
- Add more white space above the hero image (before selecting Dev/design) it's getting cut off and the content isn't centered in it per the marvel
 

Tablet:
- The animation doesn't cover the full screen it's pushed to the left
- When the user presses developer or Designer the background animation disappears (background is white) and then arrow animation is incorrect. (see screenshot posted in the previous message)
- "Show us what you made of" title and content should be left-aligned
- "Ideal candidate" title & body should be left-aligned
- "And can meet the following prerequisites:" title and body should be left-aligned 
- Form fields should be slimmer and centered per the marvel (title as well)  (Desktop & tablet) https://marvelapp.com/57djie4/screen/62585458

Mobile:
- The "Apply Now" button is not centered
- The Design/Dev animation (after selecting the dev/design button) needs to be lowered. also on chrome, the animation needs to be centered

Form:
- Change the design form to say "Portfolio" instead of profile & update placeholder text to say "your Portfolio"
- add "*" to the end of Resume/CV

**Comment by Fdom92** [comment]
Added fix for:

- Change the language under normal interview suck for the design side. that section should read: "At OpenForge, we believe that sometimes actions speak louder than words, so as part of your interview assignment, we’d like to challenge you to design 3 pages of an app of your choosing."
- Have "Apply Now" scroll less (should show "submit your application" )
- Add more white space above the hero image (before selecting Dev/design) it's getting cut off and the content isn't centered in it per the marvel
- "Show us what you made of" title and content should be left-aligned
- "Ideal candidate" title & body should be left-aligned
- "And can meet the following prerequisites:" title and body should be left-aligned
- The "Apply Now" button is not centered
- Change the design form to say "Portfolio" instead of profile & update placeholder text to say "your Portfolio"
- add "*" to the end of Resume/CV

**Comment by Wholloran** [comment]
Asset Corrections:
- "We want to put you to the test" needs asset changed (this asset changes based on dev/design)
- "Are you prepared" Needs asset changed (this asset changes based on dev/design)
- "Show us what you are made of" is still showing both dev & design asset, should only show the one animation the page is for (dev/design) https://marvelapp.com/3f8d0id/screen/63248698
- "The ideal candidate can" needs asset changed (this asset changes based on dev/design)

**Mobile:**
- The Design/Dev animation (after selecting the dev/design button) needs to be lowered. also on chrome, the animation needs to be centered

**Tablet:**
- The animation doesn't cover the full screen it's pushed to the left
- When the user presses developer or Designer the background animation disappears (background is white) and then arrow animation is incorrect. (see screenshot posted in the previous message) (This is on the physical iPad in the office)
- Form fields should be slimmer and centered per the marvel (title as well) (Desktop & tablet) https://marvelapp.com/57djie4/screen/62585458

**Comment by Fdom92** [comment]
Added fix for:

- "We want to put you to the test" needs asset changed (this asset changes based on dev/design)
- "Are you prepared" Needs asset changed (this asset changes based on dev/design)
- "The ideal candidate can" needs asset changed (this asset changes based on dev/design)
- The animation doesn't cover the full screen it's pushed to the left
- Form fields should be slimmer and centered per the marvel

**Comment by Fdom92** [comment]
Ready for other round of QA.

- When the user presses developer or Designer the background animation disappears (Can not reproduce)
- Show us what you are made of" is still showing both dev & design asset (Need this animations)
- The Design/Dev animation (after selecting the dev/design button) needs to be lowered (Might need new animation slowered)

**Comment by Wholloran** [comment]
- Show us what you are made of" is still showing both dev & design asset
assets in this folder --> https://drive.google.com/drive/folders/1aZh0O-sXIZ8cGCCz3su6F6ifJPnWgloQ

Desktop:
- "Design Master" / "Code Master" icon needs to be smaller (Desktop) (this is on the first page, the second page this asset looks fine)

Mobile: 
- The Design/Dev animation (after selecting the dev/design button) needs to be lowered (Mobile) (I'm not worried about the speed of the animation. Just need the position corrected)

Form:
- Remove the extra space between Portfolio / Github & "what makes you unique in the form

**Comment by Wholloran** [comment]
QA Passes

File: src/pages/app-opportunities/app-opportunities.scss [hunk 1]
```diff
@@ -1,43 +1,252 @@
...
+      justify-content: center;
+      align-items: center;
+      img {
+        width: 50px;
+      }
     }
 
     h2 {
-      color: $white;
+      color: #121212;
```
Comment (luiskcs89) [inline src/pages/app-opportunities/app-opportunities.scss]: should we use variables for the colors in this file?

**Pull request merged**